driver/objgen/objgen.go

```diff
@@ -21,6 +23,13 @@ type Output struct {
 }
 
 func Get(vars map[string]string) (*Output, error) {
+	return GetTemplate(vars, SecretsTemplate, CRDTemplate, RbacTemplate, CSITemplate)
+}
+
+func GetTemplate(vars map[string]string, template ...string) (*Output, error) {
```

Comment:
Ia there a need to export this function?

Reply:
It's exported because new command uses `GetTemplate` to render only selected templates.